packages/core/admin/admin/src/content-manager/components/BlocksInput/Blocks/Image.tsx

```diff
@@ -162,6 +161,12 @@ const imageBlocks: Pick<BlocksStore, 'image'> = {
     },
     matchNode: (node) => node.type === 'image',
     isInBlocksSelector: true,
+    handleEnterKey(editor) {
+      Transforms.insertNodes(editor, {
+        type: 'paragraph',
+        children: [{ type: 'text', text: '' }],
+      });
```

Comment:
if we are adding empty block on enter, then we should remove empty block added on creation, otherwise it adds 2 empty blocks of no use.